nl/mail/chrome/messenger/messengercompose/messengercompose.dtd

```diff
@@ -380,33 +380,33 @@ entities below were automatically added by l10nmerge
 -->
 
 <!ENTITY pbTopicContextMenu-assignColor.accesskey   "c">
-<!ENTITY pbTopicContextMenu-delete.label       "Delete">
+<!ENTITY pbTopicContextMenu-delete.label       "Verwijderen">
 <!ENTITY pbTopicContextMenu-delete.accesskey   "d">
-<!ENTITY pbTopicContextMenu-assignColor.label  "Set Topic Color">
+<!ENTITY pbTopicContextMenu-assignColor.label  "Stel kleur onderwerp in">
 
 
 <!--
 entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY pbStatsWords.label "words:">
+<!ENTITY pbStatsWords.label "woorden:">
 <!ENTITY pbSourceView.accesskey "d">
 <!ENTITY pbSourceViewThemes.accesskey "T">
 <!ENTITY pbStatsChars.label "characters:">
 <!ENTITY formattingToolbar.label "Toolbar">
-<!ENTITY pbShowCompositionGoalsToolbarCmd.label "Goal Status Bar">
-<!ENTITY pbStatsTime.label "time:">
-<!ENTITY pbSourceViewThemes.label "Theme">
-<!ENTITY compositionGoals.tooltip "Show or hide the Composition Goals status bar">
-<!ENTITY switchBetweenNormalAndCode.tooltip "Switch between Normal and Code views">
-<!ENTITY compositionGoals.label "Goals">
+<!ENTITY pbShowCompositionGoalsToolbarCmd.label "Doelstatusbalk">
+<!ENTITY pbStatsTime.label "Tijd:">
+<!ENTITY pbSourceViewThemes.label "Thema">
+<!ENTITY compositionGoals.tooltip "Toon of verberg de Compositiondoelen statusbalk">
```

Comment:
@kikikir I'm confused on what the right thing to do is here.  Are you saying that I should replace ''Compositiondoelen'' with ''Composition'',  like this:

Toon of verberg de Composition statusbalk